freeCodeCamp/freeCodeCamp
# fix(ui-components): button component styles

Checklist:






This PR:
- Fixes styling issues in the Button component  (resolves #52115)
- ~Exports the Button component from `ui-components` for consumption~ (hold off, see https://github.com/freeCodeCamp/freeCodeCamp/pull/52116#discussion_r1374460588)

Note: It's probably easier to review the PR with Hide Whitespace Changes enabled.

## Steps to test

- Apply the change in #52094
- Run:
  ```cmd
  cd tools/ui-components
  pnpm run storybook
  ```   
- Check the Button stories

## Screenshots

| Issue | Before | After |
| --- | --- | --- |
| Button has incorrect border style | <img width="349" alt="Screenshot 2023-10-26 at 11 48 43" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/ecb85481-9aab-4bce-b9d1-96580f34f95a"> | <img width="466" alt="Screenshot 2023-10-26 at 14 41 44" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/741b7de8-5709-47b7-ae7b-67f063361b88"> |
| Button link (a link that looks exactly like a button) doesn't have a border | <img width="749" alt="Screenshot 2023-10-26 at 11 46 38" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/0ac81ac1-6dfe-45d9-ae3a-eb5fd5c22f09"> | <img width="615" alt="Screenshot 2023-10-26 at 14 43 44" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/ea31e4ab-d495-4ef3-9587-10c311fd1617"> |
| Download link is not rendered correctly on Storybook | <img width="775" alt="Screenshot 2023-10-26 at 11 44 53" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/cb3295ab-5d55-488b-8116-1599907d8853"> | <img width="619" alt="Screenshot 2023-10-26 at 14 44 22" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/8e326983-5d2b-4e31-86b4-27be01b02223"> |
| Disabled button has a focus outline on click | <img width="229" alt="Screenshot 2023-10-26 at 14 33 31" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/1c9885e5-81a6-4db9-aa86-c5c6735031d2"> | <img width="405" alt="Screenshot 2023-10-26 at 14 45 03" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/d8c53fe7-58fa-45df-8c4c-2d3c0c3e7cf7"> |
